schema/README.md

```diff
@@ -0,0 +1,33 @@
+# JSON schema
+
+## Overview
+
+This directory contains the [JSON Schema](http://json-schema.org/) for
+validating the `config.json` of this container runtime specification.
+
+The layout of the files is as follows:
+* schema.json - the primary entrypoint for the whole schema document
+* schema-linux.json - this schema is for the Linux-specific sub-structure
+* defs.json - definitions for general types
+* defs-linux.json - definitions for Linux-specific types
+* validate.go - validation utility source code
+ 
```

Comment:
`git show --check origin/pr/360` highlights some trailing whitespace on this line.  I had a commit for checking that automatically in #182 (bdaf4467e), if you want to port it to vbatts/git-validation.